tools/ManifestSignerClient/ManifestSignerClient/CertificateUtil.cs

```diff
@@ -49,40 +51,34 @@ static public string GetBase64CertContent(string certPath)
                 throw new Exception("Could not decode PEM signer certificate - invalid base64 section", e);
             }
         }
-        static public ECDsa CreateECDsaFromPrivateKey(string SignerKeyPath)
+
+       
+
+        public static byte[] GetPrivateKeyFromPem(string keyPath, string algoStr)
```

Comment:
Nit pick. Does this need to be public? And same for the other two create cert functions. Main program seems only need the base64content and json signature.